servers/rendering/renderer_rd/shaders/resolve.glsl

```diff
@@ -58,6 +58,117 @@ void main() {
 
 #else
 
+#if 1
+
+	vec4 group1;
+	vec4 group2;
+	vec4 group3;
+	vec4 group4;
+	int best_index = 0;
+
+	//2X
+	group1.x = texelFetch(source_depth, pos, 0).r;
+	group1.y = texelFetch(source_depth, pos, 1).r;
+
+	//4X
+	if (params.sample_count <= 4) {
```

Comment:
Shouldnt these be >= ?

Right now a sample count of 2 will run all the conditions, while a sample count of 16 only uses the last

Reply:
Agreed.